spec/requests/documents_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+
+RSpec.shared_examples 'download evidence document' do
+  let(:document) { create :document, external_user: document_owner }
+  let(:document_owner) { external_user }
+  let(:test_url) { 'https://example.com/document.doc#123abc' }
+
+  before do
+    allow(Document).to receive(:find).with(document.to_param).and_return(document)
+    allow(document.document.blob).to receive(:service_url).and_return(test_url)
+    allow(document.converted_preview_document.blob).to receive(:service_url) { |args|
+      "#{test_url}?response-content-disposition=#{args[:disposition] || 'inline'}"
+    }
+  end
+
+  context 'when the document is owned by the logged in user' do
+    it { is_expected.to redirect_to "#{test_url}?response-content-disposition=#{expected_disposition}" }
```

Comment:
It will be in the response headers that comes back from S3 so we cannot check that. The headers in the response is set using the parameters in the url, like this. See https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html